app/components/layout/Navigation/index.js

```diff
@@ -21,7 +21,8 @@ class Navigation extends React.Component {
   static propTypes = {
     organization: React.PropTypes.object,
     viewer: React.PropTypes.object,
-    relay: React.PropTypes.object.isRequired
+    relay: React.PropTypes.object.isRequired,
+    assumed: React.PropTypes.bool
```

Comment:
I just added something super basic like you said.

I wanted to add it to `Viewer` but I couldn't get access to the session from `Tenant` :|